api/v1alpha1/etcdcluster_types.go

```diff
@@ -13,7 +13,7 @@ type EtcdClusterSpec struct {
 	Replicas *int32 `json:"replicas"`
 	// Storage is the configuration of the disks and mount points of the Etcd
 	// peers.
-	Storage *EtcdPeerStorage `json:"storage,omitempty"`
+	Storage EtcdPeerStorage `json:"storage,omitempty"`
```

Comment:
Is this a required field then? If so, also remove `omitempty` (it has no effect on non pointer-like types)

Reply:
Yep, I've removed omitempty.